.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 defaults: &defaults
   docker:
   - image: golang:1.11
-  working_directory: /go/src/github.com/kubermatic/cluster-api-provider-digitalocean
+  working_directory: /go/src/sigs.k8s.io/cluster-api-provider-digitalocean
```

Comment:
I think this kubernetes-sigs repo first needs to enabled in circle ci, right? Ref: https://github.com/kubernetes/test-infra/pull/9938/files#r228809144

@xmudrii If you are an admin, can you do that?